.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,59 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI Apple? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
```

Comment:
this might be hard. some of the design figma is internal only and external figma links are significantly behind the current version...

Reply:
This would be if a request for a new component was coming in externally. If it's coming internally, they can reach out either through Design or our internal teams. I can add a bit there saying `Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Apple Teams channel.`